parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -401,6 +372,21 @@ write_files:
     runtime-endpoint: unix:///run/containerd/containerd.sock
     #EOF
 
+- path: /etc/systemd/system/ensure-no-dup.service
+  permissions: "0644"
+  encoding: gzip
+  owner: root
+  content: !!binary |
+    {{GetVariableProperty "cloudInitData" "ensureNoDupEbtablesService"}}
+
+- path: /opt/azure/containers/ensure-no-dup.sh
+  permissions: "0755"
+  owner: root
+  encoding: gzip
+  content: !!binary |
+    {{GetVariableProperty "cloudInitData" "ensureNoDupEbtablesScript"}}
+{{end}}
```

Comment:
moved teleport outside this {{end}}, kept the networking bits inside because docker bridge networking is different